source/brailleDisplayDrivers/hidBrailleStandard.py

```diff
@@ -220,22 +220,39 @@ def display(self, cells: List[int]):
 				"br(hidBrailleStandard):panRight",
 				"br(hidBrailleStandard):rockerDown",
 			),
-			"braille_previousLine": ("br(hidBrailleStandard):space+dot1",),

```

Comment:
should we still have commands from previous and next line?

Reply:
braille_nextLine and kb_downArrow generally do the same thing, at least by default. If however you tether braille to the review cursor, then braille_nextLine will move the review cursor up and down by line, but I think this is a fairly rare usecase, and certainly none of the braille manufacturers I'm talking with seem to care. In future we could consider some kind of toggle command on the braille display to switch between panning and moving by line or something. But it seems the expectation is that joystick / dpad acts like pressing the arrow keys on the keyboard.